package.json

```diff
@@ -18,7 +18,9 @@
     "has-flag": "^5.0.1",
     "is-valid-path": "^0.1.1",
     "json-cycle": "^1.3.0",
-    "lodash.memoize": "^4.1.2"
+    "lodash.memoize": "^4.1.2",
+    "newman": "^5.3.2",
+    "newman-reporter-testomatio": "file:packages/newman-reporter-testomatio/newman-reporter-testomatio-0.0.1.tgz"
```

Comment:
No, this package should not depend on newman-reporter-testomatio, 

The connection should happen otherwise

newman-reporter-testomatio should depend on @testomatio/reporter

Reply:
Remove dependency on newman and neman reporter from the main package as it creates circular dependency